### ISSUE TITLE
MLP: Create plots, write report

### ISSUE THREAD

**Issue by hinderling** [hinderling/pattern_recognition#6]
• Output as PDF report / ReadMe
   • Plot showing error-rate on training and validation set with respect to the training epochs
   • Accuracy on the test set with optimized parameters
   • Average accuracy during cross-validation for all investigated kernels (e.g. linear and RBF) and all parameter values (e.g. C and γ ).

**Comment by hinderling** [comment]
make plot for score as function of iterations
make plot for score as function of iterations with shuffled labels
